### ISSUE TITLE
[FEATURE] Support multi-table truncate.

### ISSUE THREAD

**Issue by onukristo** [JSQLParser/JSqlParser#1821]
### Grammar or Syntax Description

JSQLParser fails to parse Postgres queries with multi-parameter "TRUNCATE" statement.

### SQL Example

```sql
TRUNCATE TABLE transfer_state, transfer_state_change_log
```

```
net.sf.jsqlparser.parser.ParseException: Encountered unexpected token: "," ","
    at line 1, column 30.
```

### Additional context
JSQLParser version: 4.6.
I also tested with 4.7 SNAPSHOT
Postgres sql.

https://www.postgresql.org/docs/12/sql-truncate.html

The multi-table truncate statement is necessary in case the two tables are referencing each other by some constraints. In that case Postgres returns an error if you try to truncate each table individually.

**Comment by manticore-projects** [comment]
Greetings.

This is Postgres specific and not standard compliant. I would like to recommend writing standard compliant SQL.

If you still need support for any RDBMS specific feature then either provide or sponsor an implementation.

**Comment by manticore-projects** [comment]
Should be solved via #2048
